microcloud/test/suites/basic.sh

```diff
@@ -822,3 +822,135 @@ test_auto() {
+  export IPV4_START="10.1.123.100"
+  export IPV4_END="10.1.123.254"
+  export DNS_ADDRESSES="10.1.123.1,8.8.8.8"
+  export IPV6_SUBNET="fd42:1:1234:1234::1/64"
+
+  reset_systems 3 3 3
+  echo Create a MicroCloud that re-uses an existing service
+  export REUSE_EXISTING_COUNT=1
+  export REUSE_EXISTING="add"
+  lxc exec micro02 -- microceph cluster bootstrap
+  microcloud_interactive | lxc exec micro01 -- sh -c "microcloud init > out"
+  services_validator
+
+  reset_systems 3 3 3
+  echo Create a MicroCloud that re-uses an existing service on the local node
+  lxc exec micro01 -- microceph cluster bootstrap
+  microcloud_interactive | lxc exec micro01 -- sh -c "microcloud init > out"
+  services_validator
+
+  reset_systems 3 3 3
+  echo Create a MicroCloud that re-uses an existing MicroCeph and MicroOVN
+  export REUSE_EXISTING_COUNT=2
+  export REUSE_EXISTING="add"
+  lxc exec micro02 -- microceph cluster bootstrap
+  lxc exec micro02 -- microovn cluster bootstrap
+  microcloud_interactive | lxc exec micro01 -- sh -c "microcloud init > out"
+  services_validator
+
+  reset_systems 3 3 3
+  echo Create a MicroCloud that re-uses an existing MicroCeph and MicroOVN on different nodes
+  lxc exec micro02 -- microceph cluster bootstrap
+  lxc exec micro03 -- microovn cluster bootstrap
+  microcloud_interactive | lxc exec micro01 -- sh -c "microcloud init > out"
+  services_validator
+
+  reset_systems 3 3 3
+  echo Create a MicroCloud that re-uses an existing existing service with multiple nodes from this cluster
+  export REUSE_EXISTING_COUNT=1
+  export REUSE_EXISTING="add"
+  lxc exec micro02 -- microceph cluster bootstrap
+  token="$(lxc exec micro02 -- microceph cluster add micro01)"
+  lxc exec micro01 -- microceph cluster join "${token}"
+  microcloud_interactive | lxc exec micro01 -- sh -c "microcloud init > out"
+  services_validator
+
+  reset_systems 3 3 3
+  echo Create a MicroCloud that re-uses an existing existing service with all nodes from this cluster
+  lxc exec micro02 -- microceph cluster bootstrap
+  token="$(lxc exec micro02 -- microceph cluster add micro01)"
+  lxc exec micro01 -- microceph cluster join "${token}"
+  token="$(lxc exec micro02 -- microceph cluster add micro03)"
+  lxc exec micro03 -- microceph cluster join "${token}"
+  microcloud_interactive | lxc exec micro01 -- sh -c "microcloud init > out"
+  services_validator
+
+  reset_systems 4 3 3
+  echo Create a MicroCloud that re-uses an existing existing service with foreign cluster members
+  lxc exec micro04 -- snap disable microcloud
+  lxc exec micro02 -- microceph cluster bootstrap
+  token="$(lxc exec micro02 -- microceph cluster add micro04)"
+  lxc exec micro04 -- microceph cluster join "${token}"
+  microcloud_interactive | lxc exec micro01 -- sh -c "microcloud init > out"
+  services_validator
+  validate_system_microceph micro04 1
+
+  reset_systems 3 3 3
+  echo Fail to create a MicroCloud due to an existing service if --auto specified
```

Comment:
This is one good example where double quoting the string passed to `echo` would help the text highlighting because this `if` shows as a command instead of text. :)